src/slide-potentiometer-element.stories.ts

```diff
@@ -14,12 +14,11 @@ export default {
 };
 
 const Template = ({ value }) =>
-  html` <wokwi-slide-potentiometer
+  html`<wokwi-slide-potentiometer
     value=${value}
     @button-press=${action('button-press')}
     @button-release=${action('button-release')}
-  >
-  </wokwi-slide-potentiometer>`;
+  />`;
```

Comment:
Can we please add another story featuring the potentiometer rotated? We can replace the 2nd "Large" story with "Rotated" or "Portrait" story that wraps the element within a transformed div

Reply:
I added a portrait story

Reply:
Awesome!